README.md

```diff
@@ -242,3 +242,4 @@ platform.withFeatures(notifyOnSuccess('It worked!'), notifyOnError('It failed!')
 
 -   You can either manually write a commit message that follows the convention using your favorite method.
 -   Or you can run `npm run commit-cli`. It will prompt you some questions about the nature of your changes, then it will commit your staged changes along with a generated message that follows our convention.
+-   Commits containing breaking changes need to be markeed as such in the commit footer. See [BREAKING CHANGE convention](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-footer).
```

Comment:
Just a misspelling, might as well fix since it's new.
```suggestion
-   Commits containing breaking changes need to be marked as such in the commit footer. See [BREAKING CHANGE convention](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-footer).
```

Also a number of commits are showing up, maybe needs rebased or something?

Reply:
lol this whole thing has fried my brain

Reply:
the old commits have already been squash/merged and this branch is up to date with master. it's just showing them because those commit ids haven't been merged due to the squash.